include/core/parameters.h

```diff
@@ -238,6 +238,9 @@ namespace Parameters
     // Enable velocity post-processing
     bool calculate_average_velocities;
 
+    // Enable reynolds stress post-processing
+    bool calculate_reynolds_stress;
+
```

Comment:
I wonder if we should really add this. I mean, when you calculate averages you will most likely calculate_average_velocities we will also want the Reynolds stresses. I'm not sure we need an additional parameter for this shenanigan :)